packages/terra-application-header-layout/src/ApplicationHeaderLayout.jsx

```diff
@@ -79,10 +85,21 @@ const ApplicationHeaderLayout = ({
     );
   }
 
+  const skipToContent = () => {
+    document.getElementById('terra-main').focus();
+  };
+
+  const skipToContentButton = (
+    <button type="button" onClick={skipToContent} className={cx('skip-content')}>
```

Comment:
Should we be using Terra-button for this update?

Reply:
Brett and I talked. Given that this button is visually hidden most of the time, it doesn't make sense to inject a dependency requirement just for this one button.